packages/field-base/src/label-for-controller.js

```diff
@@ -5,10 +5,10 @@
  */
 
 /**
- * A controller to link an input element with a slotted `<label>` element.
+ * A controller to link a label element with a slotted `<input>` element.
  */
-export class AriaLabelController {
-  constructor(host, input, label) {
+export class LabelForController {
+  constructor(_host, input, label) {
```

Comment:
Looks like the host parameter is unused, do we have it here only to match `ReactiveController` interface?

Reply:
I'll double-check it and remove the argument if it is not required by the interface.

Reply:
Removed.